distribution/docs/src/main/resources/content/_running/catalog-commands.adoc

```diff
@@ -0,0 +1,146 @@
+:title: Catalog Commands
+:type: subMaintaining
+:status: published
+:parent: Available System Console Commands
+:summary: Catalog commands available.
+:order: 00
+
+== {title}
+
+[WARNING]
+====
+Most commands can bypass the Catalog framework and interact directly with the Catalog provider if given the `--provider` option, if available.
+No pre/post plugins are executed and no message validation is performed if the `--provider` option is used.
+====
+
+.[[catalog_command_descriptions]]Catalog Command Descriptions
+[cols="1m,9a" options="header"]
+|===
+
+|Command
+|Description
+
+|catalog:describe
+|Provides a basic description of the Catalog implementation.
+
+|catlog:dump
+|Exports metacards from the local Catalog. Does not remove them. See <<_date_filtering_options,date filtering options>> below.
+
+|catalog:envlist
+|[IMPORTANT]
+====
+Deprecated as of ddf-catalog 2.5.0. Please use `platform:envlist`.
+====
+
+Provides a list of environment variables.
+
+|catalog:ingest
+|Ingests data files into the Catalog.
+
+|catalog:inspect
+|Provides the various fields of a metacard for inspection.
+
+|catalog:latest
+|Retrieves the latest records from the Catalog based on the Metacard.MODIFIED date.
+
+|catalog:migrate
+|Allows two `CatalogProvider` s to be configured and migrates the data from the primary to the secondary.
+
+|catalog:range
+|Searches by the given range arguments (exclusively).
+
+|catalog:remove
+|Deletes a record from the local Catalog.
+
+|catalog:removeall
+|Attempts to delete all records from the local Catalog.
+
+|catalog:replicate
+|Replicates data from a federated source into the local Catalog.
+
+|catalog:search
+|Searches records in the local Catalog.
+
+|catalog:spatial
+|Searches spatially the local Catalog.
+
+|catalog:transformers
+|Provides information on available transformers.
+
+|catalog:validate
+|Validates an XML file against all installed validators and prints out human readable errors and warnings.
+
+|===
+
+.catalog:dump Options
+The `catalog:dump` command provides selective export of metacards based on date ranges.
+The `--created-after` and `--created-before` options allow filtering on the date and time that the metacard was created, while `--modified-after` and `--modified-before` options allow filtering on the date and time that the metacard was last modified (which is the created date if no other modifications were made).
+These date ranges are exclusive (i.e., if the date and time match exactly, the metacard will not be included).
+[[_date_filtering_options]]The date filtering options (`--created-after`, `--created-before`, `--modified-after`, and `--modified-before`) can be used in any combination, with the export result including only metacards that match all of the provided conditions.
+
+If no date filtering options are provided, created and modified dates are ignored, so that all metacards match.
+
+.Date Syntax
```

Comment:
What is the purpose of this section? Is this syntax only used for some specific `catalog` commands?

Reply:
That's a good question